.platform/routes.yaml

```diff
@@ -37,6 +37,10 @@ https://{default}/:
       # Reorganized August 2020
       "^/gettingstarted/gettingstarted(.*)": { "to": "/gettingstarted/introduction$1", "code": 301, "regexp": true }
 
+      # Reorganized September 2020
+      "/golive/steps/dns.html": { "to": "/domains/in-depth/dns.html", "code": 301, "prefix": false}
+      "/golive/steps/subdomains.html": { "to": "/domains/in-depth/subdomains.html", "code": 301, "prefix": false}
```

Comment:
Didn't Chad already add these in another branch?

Reply:
I merged in this branch.